### ISSUE TITLE
Ship capturing crew dispersal

### ISSUE THREAD

**Issue by ConfusingDalek** [endless-sky/endless-sky#2917]
When capturing a ship, some of your crew leave to get on the other ship and pilot it. This is expected. However, even when there are enough crew to fly your ship already, enough leave your ship that both ships do not have sufficient crew. At the very least, it would be good if your flagship retained enough crew for control if at all possible.

**Comment by KyroFirehammer** [comment]
Agreed

**Comment by tehhowch** [comment]
This change would make capturing more powerful, not more balanced. If anything, the more damaged ship should get more crew than the flagship so it can more easily avoid danger.

Currently the game is fair: when the total available crew is insufficient to fully crew both ships, the crew is proportionally divided among the two ships: [Line 1850](https://github.com/endless-sky/endless-sky/blob/master/source/Ship.cpp#L1850)
If 60 crew is required (50 and 10), and you have 50, you give them `50 * 10 / 60` = 8, resulting in an 8/10 (80% crewed) and a 42/50 (84% crewed), and both ships will maneuver roughly equivalently.

In other words, I believe this issue is a candidate for the `unlikely` label

**Comment by marphod** [comment]
I could see a configurable preference to bias the crew to the flagship, or a preference of 'leave flagship as full as possible', but both make capturing better.  The changes also wouldn't make sense without changing how insuffcient crews affect piloting -- i think it would require ships to have a minimum threshold (that is 'this ship requires 10 crew to work even intermittently, and 50 crew to work 100% of the time') rather than the current 'any crew will do, just intermittently' current model.

**Comment by Hadron1776** [comment]
> preference of 'leave flagship as full as possible'

Which basically means, "don't transfer any crew"

**Comment by ConfusingDalek** [comment]
Also, at whatever point this should probably be closed, let me know, because I'm kinda new to this.

**Comment by marphod** [comment]
> Which basically means, "don't transfer any crew"

Well, transfer not a fair distribution, but such that the new ship has only enough crew to operate intermittently (which would require the change I mentioned)

**Comment by endlesssteve** [comment]
Isn't there the risk here you end up capturing a ship with zero crew. A bit pointless.

**Comment by jafdy** [comment]
If the ship is automated then you don't need to control it with crew.
If the ship is not automated then you can't attack with less than 2 crew (one for each ship).

**Comment by KyroFirehammer** [comment]
Or possibly a way to transfer after capping some crew from a third ship to fill up the missing crew on the flagship but only if it wouldn't put ship three into a less than 30% crew ( in that case it would get as close without putting it under 30% ). For example if you flagship has 10 crew min and you cap a ship losing 5 crew then a third ship could transfer over crew to where you can land without it taking forever and a day.

**Comment by KyroFirehammer** [comment]
Keep in mind though this would lessen the OPness of the bactrian and would lessen it as a nessecity to capping anything greater than a marauder falcon

**Comment by Bladewood** [comment]
This does not help with the bactrian issue *at all*.

**Comment by KyroFirehammer** [comment]
It probably would because ships with better find but less crew could still be useful in capping and thus making the navy's large crew space less of an issue

**Comment by tehhowch** [comment]
It does not help because the Bactrian benefits even more - as you say, if the "3rd party helper ship" can only go to 30% crew, wouldn't you want that 70% that can be transferred to be the maximum amount possible? Then the only helper ship that makes sense is the one that can have the most bunks, since that lets you transfer the most crew, and now you have players using multiple Bactrians to support their primary flagship Bactrian instead of just one.

Required crew sharing is not going to be a mechanic that changes anytime soon in the base game. Mod creators are free to spoil the experience by making every capturable ship have 0% self destruct and 0 required crew, but the base game will offer some semblance of parity among income generation methods.

**Comment by gunqqer** [comment]
I'm fine if you mod your own game to make things easier, e.g. changing the self destruct rate. You are free to do that. I am fine with you editing your own save file and giving yourself what ever you want. Doesn't mean we'll do it for everyone. Maybe in the future when we have difficulty levels we'll have a sandboxy/easy mode

**Comment by marphod** [comment]
>  the base game will offer some semblance of parity among income generation methods.

Oh?  When will this change be happening?  ;)

(As it stands, Board&Capture and Board&Pillage outpace pretty much everything else at endgame.  People Transport missions are better than Cargo missions which are better than simple freight. Tributes are steady but pathetic vis-a-vis actual costs. Anything is better than most pirate missions, which often have negative EV (fines significantly higher than payouts, high security systems, and needing to get significant amounts of interference paneling on every ship in the fleet as the control over where cargo goes is minimal).  No idea how well mining pays as i find it terminally dull.)

**Comment by tehhowch** [comment]
As I said, a semblance ;) I'm no stranger to the strength of boarding tactics, which is why I can often be found explaining why some change that makes boarding easier / better is unlikely or poorly thought-out (at best).

(I believe exponential barge is better at income generation than board and cap/plunder, but its own self-balancing characteristic - being tediously boring - has prevented me from bothering with it :grinning: )

**Comment by gunqqer** [comment]
Currently we need to nerf boarding (correctly), not buffing it. 

(And a problem with exponential barge is that with exponents my computer takes a hit 😃

**Comment by endlesssteve** [comment]
Although a sensible idea I think depreciation failed to balance boarding. You just need to board 4 times more ships which is not hard. At the moment you can kill thousands of crew members boarding ships with no penalty. I even feel guilty some times.

**Comment by ghoulavenger** [comment]
>As it stands, Board&Capture and Board&Pillage outpace pretty much everything else at endgame. People Transport missions are better than Cargo missions which are better than simple freight.

I haven't been paying close attention to this issue or I might have replied sooner, but cargo missions are only better than freight if you have a small fleet. Freight actually only has a limit to how much cargo you can haul. I have a file with about 100 deep rivers with jump drives, which nets me 25-30 million a jump doing nothing but trading cargo. And it's not like I can't expand the size of that fleet either, all I would need to do is to farm up a few jump drives first (they're small, and with the fast forward button shouldn't be too hard to acquire from the Korath, or loot from the Pug). At my best attempt at capturing ships, I can make about 15-20 million in pirate territory on a full bactrian crew with androids, which isn't substantially better than just capturing Korath raiders (which go for about 12.5m) or looting triple plasma cores (which sell for 5m a pop). In either case, pushing past 20 million on a run is difficult, and that includes two jumps (unless you're sitting on top of a shipyard/outfitter). I suppose if you were jettisoning the triple plasma cores to other ships with huge cargo capacities you might be able to make more on a per jump basis that way, but it still costs a lot more time than a single jump turn around. Freight might not the fastest way to riches, but it is the most extensible way.

>(I believe exponential barge is better at income generation than board and cap/plunder, but its own self-balancing characteristic - being tediously boring - has prevented me from bothering with it 😀 )

Not in the early game (make like 10k a jump or something, when you can just capture a sparrow for over 100k instead), and if you were going to do it in the late game you'd need to do it outside of human space, because you'd lose too many ships to pirates otherwise. If you were going this route you'd probably want to do at least exponential freighter, to help compensate for risk. Personally I'm a fan of deep rivers, especially after they got buffed -- but their lack of turrets still make them less desirable for human space (no AM coupled with a slow turning speed with only forward firing guns means they aren't exactly battle worthy, only their toughness keeps them afloat), although in mass they can still tear down leviathans pretty easily, and with jump drives can skip most of the combat anyway. A more casual player might want to go for behemoths or bulk freighters instead, possibly even stripped down bactrians stuffed with boxwings.

If you were going to extend this strategy straight from pirate hunting though you might consider the Argosy, the Headhunter, and maybe the Clipper (no turrets so no AM, otherwise solid choice).
